src/rsvc/srv.c

```diff
@@ -956,21 +965,17 @@ ds_rsvc_dist_start(enum ds_rsvc_class_id class, d_iov_t *id,
 		in->sai_flags |= RDB_AF_BOOTSTRAP;
 	in->sai_size = size;
 	in->sai_ranks = (d_rank_list_t *)ranks;
-
-	rc = dss_rpc_send(rpc);
-	if (rc != 0)
-		goto out_mem;
-
+	dss_rpc_send(rpc); /* Ignore return code; errors are detected below. */
 	out = crt_reply_get(rpc);
-	rc = out->sao_rc;
-	if (rc != 0) {
-		D_ERROR(DF_UUID": failed to start%s %d replicas\n",
-			DP_UUID(dbid), create ? "/create" : "", rc);
-		ds_rsvc_dist_stop(class, id, ranks, create);
+	rsvc_corpc_incr_cb(in->sai_ranks, rpc->cr_ep.ep_rank, out->sao_rc,
+			   &nstarted);
+
+	if (!nstarted || (ranks && nstarted < ranks->rl_nr)) {
```

Comment:
it seems ranks will never be NULL.   so should we simplify the check here?   Hmm, do you need stop those started replicas if this failure happen.

TBH, this bcast process looks a bit tricky. so it sends the RPC to non-svc replica nodes only because corpc only has cart group interface?